projects/spidermonkey-ufi/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+MAINTAINER cdiehl@mozilla.com
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    autoconf2.13 \
+    python \
+    libc++1 \
```

Comment:
I'm not going to block this if it works. But this seems like a bit of a crazy hack to me (though I don't really know much about linking).
Is it not possible to build these fuzzers with libc++ linked statically? No other project seems to have this issue.

If not, we may be able to solve this on the oss-fuzz side by building the shared versions of libc++ and libc++abi (which we explicitly opt-out of [here](https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-clang/checkout_build_install_llvm.sh#L86)). So you don't need to install one from apt.

@morehouse WDYT about this?